src/modules/transaction/manager/recentTransactionManager.js

```diff
@@ -1,22 +1,5 @@
 // istanbul ignore file
 import { withTranslation } from 'react-i18next';
-import withData from 'src/utils/withData';
-import { selectActiveTokenAccount } from 'src/redux/selectors';
 import RecentTransaction from '../components/RecentTransactions/RecentTransactions';
-import { getTransactions } from '../api';
 
-export default withData({
-  transactions: {
-    apiUtil: (network, params) => getTransactions({ network, params }),
-    getApiParams: (state) => {
-      const wallet = selectActiveTokenAccount(state);
-      const address = wallet.summary?.address;
-      return {
-        address,
-        network: state.network,
-      };
-    },
-    defaultData: [],
-    transformResponse: response => response.data.splice(0, 5),
-  },
-})(withTranslation()(RecentTransaction));
+export default withTranslation()(RecentTransaction);
```

Comment:
We don't need translation here we can use 
`useTranslation` in the RecentTransaction component

Reply:
yeah, but its just out of scope of this pr, we have a different task to fix this transaction recent